### PR TITLE
more info added to mocked client connection logs

### PR DESCRIPTION
**Pull request by sancar**

### CODE REVIEW

**Comment by sancar** [comment]
run-lab-run

**Comment by sancar** [comment]
run-lab-run

**Comment by hasancelik** [comment]
run-lab-run

File: hazelcast-client/src/test/java/com/hazelcast/client/test/TestClientRegistry.java [hunk 1]
```diff
@@ -236,7 +236,7 @@ public void run() {
 
                 @Override
                 public String toString() {
-                    return "Runnable message " + clientMessage;
+                    return "Runnable message " + clientMessage + ",localAddress " + localAddress + "," + MockedClientConnection.this;
```
Comment (mdogan) [inline hazelcast-client/src/test/java/com/hazelcast/client/test/TestClientRegistry.java]: Why don't you just implement `toString()`  in `MockedClientConnection` class? In that case, you would write;

```java
return "Runnable message " + clientMessage + ", Connection: " + MockedClientConnection.this;
```

Comment (sancar) [inline hazelcast-client/src/test/java/com/hazelcast/client/test/TestClientRegistry.java]: Done.

**Comment by jerrinot** [comment]
run-lab-run

**Pull request merged**